lib/user.rb

```diff
@@ -0,0 +1,38 @@
+require 'bcrypt'
+
+class User
+  attr_reader :id, :email
+
+  def initialize(id:, email:)
+    @id = id
+    @email = email
+  end
+  
```

Comment:
Classes written very clearly. Nice and readable with short methods

Reply:
Thank you